src/components/user/user.service.ts

```diff
@@ -286,6 +286,8 @@ export class UserService {
             canReadRegion: true,
             canCreateCountry: true,
             canReadCountry: true,
+            canCreateProjectMember: true,
+            canReadProjectMembers: true,
```

Comment:
@michaelmarshall I'm not up to date on this code, but I believe this should be determined by the project not globally. Members will always be created in the context of a project and we'll check `project.team.canCreate/Read` (API) for access.